examples/add_pedestal_noise_template.xml

```diff
@@ -0,0 +1,114 @@
+            topSlopeLimit="${RemoveSpikes.topSlopeLimit}"
+            maxSpikeLength="${RemoveSpikes.maxSpikeLength}"
+        />
+        <fact.datacorrection.InterpolateTimeSeries
+                calibService="calibService"
+                dataKey="DataCalibrated"
+                dataOutputKey="DataCalibrated"
+        />
+        
+        
+    <!-- Sample Pedestal Event and calibrate it -->	
+        <fact.utils.SamplePedestalEvent prependKey="LONS_" noiseDatabase="${noiseDB}" dataFolder="${dataFolder}"
+                                        dbBinningKey="Zd" itemBinningKey="MPointingPos.fZd" binning="${noiseBinning}"
+        />
+        
+        <fact.features.UnixTimeUTC2DateTime inputKey="LONS_UnixTimeUTC" outputKey="LONS_timestamp" />
+    
+        <fact.datacorrection.DrsCalibration
+                drsKey="LONS_drspath"
+                key="LONS_Data"
+                outputKey="LONS_DataCalibrated"
+                startCellKey="LONS_StartCellData"
+        />
+
+        <fact.datacorrection.PatchJumpRemoval
+                dataKey="LONS_DataCalibrated"
+                outputKey="LONS_DataCalibrated"
+                prevEventsKey="LONS_prevEvents"
+                startCellKey="LONS_StartCellData"
+                unixTimeKey="LONS_UnixTimeUTC"
+                jumpLimit="${PatchJumpRemoval.jumpLimit}"
+        />
+
+         <fact.datacorrection.RemoveSpikes
+                dataKey="LONS_DataCalibrated"
+                outputKey="LONS_DataCalibrated"
+                startCellKey="LONS_StartCellData"
+                leftBorder="${RemoveSpikes.leftBorder}"
+                spikeLimit="${RemoveSpikes.spikeLimit}"
+                topSlopeLimit="${RemoveSpikes.topSlopeLimit}"
+                maxSpikeLength="${RemoveSpikes.maxSpikeLength}"
+        />
+
+        <fact.datacorrection.DrsTimeCalibration
+                startCellKey = "LONS_StartCellData"
+                dataKey      = "LONS_DataCalibrated"
+                outputKey    = "LONS_DataCalibrated"
+        />
+
+        <fact.datacorrection.InterpolateTimeSeries
+                calibService="calibService"
+                dataKey="LONS_DataCalibrated"
+                dataOutputKey="LONS_DataCalibrated"
+                badPixelKey="LONS_badPixel"
+                timeStampKey="LONS_timestamp"
+        />
+
+        <fact.utils.CombineDataArrays firstArrayKey="DataCalibrated" secondArrayKey="LONS_DataCalibrated"
+                                      outputKey="DataCalibrated" op="add"/>
+        
+         <fact.datacorrection.DrsCalibration
+                key="DataCalibrated"
+                outputKey="Data"
+                reverse="true"
+        />
+        <fact.utils.Remapping
+                key="Data"
+                outputKey="Data"
+                reverse="True"
+        />
+        <!--<fact.ShowViewer key="DataCalibrated" />-->
+
+        <fact.io.FITSWriter url="${output}"
+                            keys="${OUTPUT_KEYS}"
```

Comment:
could this rather be realised with wildcards and exclusions?

Reply:
Would be nice but there is no good way to exclude all keys that are not needed, we would need a better way of naming our keys first (for example all header keys should for example start the same way).

Reply:
added wildcards above